### PR TITLE
qt-svg: new package for Qt6 SVG module

### PR DESCRIPTION
**Pull request by aumuell**
Depends on #40833 

enables loading of SVG icons by providing plugin used by qt-base

### CODE REVIEW

File: var/spack/repos/builtin/packages/qt-svg/package.py
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/qt-svg/package.py:15]: I realize there is a default `qt` homepage but is it worth adding a more specific one here (i.e., `https://doc.qt.io/qt-6/qtsvg-index.html`)?

File: var/spack/repos/builtin/packages/qt-svg/package.py
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/qt-svg/package.py:20]: These versions aren't yet made available in the `qt-base` package so the concretizer will resolve that dependency to v6.5.2.  Is that desirable?  If not, you probably want to update the `qt-base` package as well or remove these versions for now.

**Review by tldahlgren — CHANGES_REQUESTED** [review]
I have confirmed the 10 version sha256 but have questions on this PR.

File: var/spack/repos/builtin/packages/qt-svg/package.py
Comment (tldahlgren) [reply var/spack/repos/builtin/packages/qt-svg/package.py:20]: Nvm.  I just saw the PR to update those so I'll add a dependencies in the description.

**Review by tldahlgren — APPROVED** [review]
I still have the question about the homepage though.

**Comment by tldahlgren** [comment]
@sethrj @wdconinc Since you both appear to be maintainers on a number of `qt` packages, do you want to comment on this one?

**Comment by sethrj** [comment]
I'll defer to @wdconinc ; he's the main qt user these days, and now he has magical merge privileges! 🎉

File: var/spack/repos/builtin/packages/qt-svg/package.py
Comment (aumuell) [reply var/spack/repos/builtin/packages/qt-svg/package.py:15]: This probably makes sense, but not just for qt-svg: all the Qt packages except for qt-base and qt-declarative could do so. Here is a draft for changing that: #40850

**Review by wdconinc — APPROVED** [review]
Both most recent version tested, and build fine on my test system (without widgets variant).

**Comment by wdconinc** [comment]
> he's the main qt user these days

I keep waiting for geant4 to support Qt6...

**Pull request merged**
